lib/modules/apostrophe-templates/views/outerLayoutBase.html

```diff
@@ -15,12 +15,20 @@
     {% endblock %}
 
     {% if data.user %}
-    <div class="apos-ui">
-      <div class="apos-context-menu-container">
-        {{ apos.pages.publishMenu({ publishMenu: data.publishMenu, page: data.page, piece: data.piece, bottom: true }) }}
-        {{ apos.pages.menu({ contextMenu: data.contextMenu, page: data.page, bottom: true })}}
-      </div>
-    </div>
+      {% block apostropheContextMenuWrapper %}
+        <div class="apos-ui">
+          {% block apostropheContextMenu %}
+            <div class="apos-context-menu-container">
+              {% block beforeApostropheContextMenu %}
+              {% endblock %}
+              {{ apos.pages.publishMenu({ publishMenu: data.publishMenu, page: data.page, piece: data.piece, bottom: true }) }}
+              {{ apos.pages.menu({ contextMenu: data.contextMenu, page: data.page, bottom: true })}}
+              {% block afterApostropheContextMenu %}
+              {% endblock %}
+            </div>
+          {% endblock %}
+        </div>
+      {% endblock %}
```

Comment:
Yes. I think I did it right to make this space more flexible in general, but it is reasonable to ask if this is even where @stuartromanek will want markup to go. Also we could approach it by registering helpers to be called or something, so it doesn't require overriding outerLayout for every project.